src/main/java/com/spotify/docker/Utils.java

```diff
@@ -64,5 +66,21 @@ public static void pushImage(DockerClient docker, String imageName, Log log)
       log.info("Pushing " + imageName);
       docker.push(imageName, new AnsiProgressHandler());
   }
+  
+  public static void removeImage(DockerClient docker, String imageName, Log log)
+      throws DockerException, InterruptedException {
+      log.info("Removing -f " + imageName);
+      try {
+          // force the image to be removed but it don't remove untagged parents 
```

Comment:
should be in fact _don't remove_, as previously was planned to be _won't_ - but it got mixed somehow... - fix is pushed already